lxc/remote.go

```diff
@@ -267,9 +267,21 @@ func (c *cmdRemoteAdd) addRemoteFromToken(addr string, server string, token stri
 		req.Password = token
 	}
 
+	// Add client certificate to trust store. Even if we are already trusted (src.Auth == "trusted"),
+	// we want to send the token to invalidate it. Therefore, we can ignore the client already trusted error.
 	err = d.CreateCertificate(req)
+	if err != nil && err.Error() != "Client is already trusted" {
```

Comment:
I don't think we should match on the error string but I'm not sure if there is a better way...

Reply:
Yes, I was looking at the status code, but it is generic 400 BadRequest. Not sure if I can wrap the error over the connection - should try that

Reply:
No it won't be possible to do that. The client recreates an `api.StatusError` based on the status code of the request and the error message. So all the wrapping is gone.

We could potentially add the error message to the `api` package but I'm not a fan of that either tbh... Lets leave for now.

Reply:
I can maybe put the error into shared and use from both `lxc` and `lxd` packages?

Reply:
Maybe I can return a 409 conflict? Since conflict is returned if user fails to add a new certificate, the same could apply when using trust token, as it is only used to add client's certificate?